src/routers/TabRouter.js

```diff
@@ -62,6 +62,15 @@ export default (
       let state = inputState;
       if (!state) {
         const routes = order.map((routeName: string) => {
+          let accessibilityLabel;
```

Comment:
Hey Matt! Thank you for reviewing this for me! 
So here is my thought process about this part. I noticed iOS and Android implemented this a bit differently. When we create a tab for Android, it create the tab using [TabBarTop](https://github.com/react-community/react-navigation/blob/master/src/views/TabView/TabBarTop.js), and TabBarTop uses [TabBar](https://github.com/react-native-community/react-native-tab-view/blob/master/src/TabBar.js) from [react-native-tab-view](https://github.com/react-native-community/react-native-tab-view) library. So that for achieving your suggestion, some changes to react-native-tab-view library is required. 
Also,  for iOS react-navigation did the magic in [TabBarButtom](https://github.com/react-community/react-navigation/blob/master/src/views/TabView/TabBarBottom.js). 
What's your suggestion?